infrastructures/kubernetes/src/test/java/org/eclipse/che/workspace/infrastructure/kubernetes/wsplugins/BrokersResultTest.java

```diff
@@ -0,0 +1,67 @@
+package org.eclipse.che.workspace.infrastructure.kubernetes.wsplugins;
+
+import org.eclipse.che.api.workspace.server.spi.InfrastructureException;
+import org.testng.annotations.Test;
+
+/**
+ * @author Alexander Garagatyi
+ */
+public class BrokersResultTest {
+  private BrokersResult brokersResult;
+
+  @Test(expectedExceptions = IllegalStateException.class)
+  public void shouldThrowExceptionOnCallingOneMoreResultAfterCallGet() throws Exception {
+    brokersResult.get();
+    brokersResult.oneMoreBroker();
+  }
+
+  @Test(expectedExceptions = IllegalStateException.class)
+  public void shouldThrowExceptionOnCallingErrorBeforeCallGet() throws Exception {
+    brokersResult.error();
+  }
+
+  @Test(expectedExceptions = IllegalStateException.class)
```

Comment:
I think it makes sense add `expectedMessagePattern` to clarify that right exception is thrown. WDYT?

Reply:
Agree